allennlp/models/archival.py

```diff
@@ -128,7 +128,11 @@ def archive_model(
 
 
 def load_archive(
-    archive_file: str, cuda_device: int = -1, overrides: str = "", weights_file: str = None
+    archive_file: str,
+    weights_file: str = None,
```

Comment:
I agree with putting the `opt_level` next to `cuda_device`, but can you keep `weights_file` at the end?  `cuda_device` is the most likely additional argument to have been passed in existing code, and moving it breaks that code.  The other keys were probably passed by name, if they were ever used at all.

You can move around the docstring parameters to match this.

Reply:
Whoops, didn't think about how that would break existing code. Fixed!